core/templates/base-components/oppia-footer.component.html

```diff
@@ -1,6 +1,37 @@
 <footer class="oppia-footer" role="contentinfo">
+  <div class="oppia-footer-newsletter-container">
+    <div class="oppia-footer-newsletter">
+      <div class="oppia-newsletter-content">
+        <h3 [innerHTML]="'I18N_FOOTER_NEWSLETTER_HEADING' | translate"></h3>
+        <p [innerHTML]="'I18N_FOOTER_NEWSLETTER_TEXT' | translate"></p>
+      </div>
+      <div class="oppia-footer-newsletter-input">
+        <label id="email"
+               [innerHTML]="'I18N_FOOTER_NEWSLETTER_EMAIL' | translate"
+        >
+        </label>
```

Comment:
```suggestion
        <label id="email">
            {{ 'I18N_FOOTER_NEWSLETTER_EMAIL' | translate }}
        </label>
```
Prefer to use interpolation, here and elsewhere.

Reply:
Done, Thanks